dissect/util/feature.py

```diff
@@ -0,0 +1,62 @@
+import functools
+import os
+from typing import Callable, Optional
+
+# Register feature flags in a central place to avoid chaos
+F_NOVICE = "novice"
```

Comment:
I'm not sure I get the purpose of a "novice" feature flag, could you give a usage example?

Reply:
The novice flag is to shield users from advanced features that might be risky. 

Usage example:

```
@feature(Feature.NOVICE, real_private_keys):
@export(record=PrivateKeyRecord)
    def private_keys(self) -> Iterator[PrivateKeyRecord]:
       print("Are you sure?....")
    .....
```

because user might end up with private keys from client, which might have consequences depending on country/jurisdiction.

Technical issues may also play a role:


```
@feature(Feature.NOVICE, real_central_av_dump):
@export(record=PrivateKeyRecord)
    def private_keys(self) -> Iterator[PrivateKeyRecord]:
        print("This can take a while... and you might acquire too much...")
    .....
```
If a user wants to obtain a central av log and the client of the user uses the same cluster for all databases, the user might end up downloading all DBs of the customer, so the user might need to check this first.